app/frontend/stores/root-store.ts

```diff
@@ -15,9 +17,11 @@ export const RootStoreModel = types
     sessionStore: types.optional(SessionStoreModel, {}),
     userStore: types.optional(UserStoreModel, {}),
     permitApplicationStore: types.optional(PermitApplicationStoreModel, {}),
+    permitClassificationStore: types.optional(PermitClassificationStoreModel, {}),
     jurisdictionStore: types.optional(JurisdictionStoreModel, {}),
-    requirementBlockStore: types.optional(RequirementBlockStore, {}),
-    requirementTemplateStore: types.optional(RequirementTemplateStore, {}),
+    RequirementBlockStoreModel: types.optional(RequirementBlockStoreModel, {}),
+    RequirementTemplateStoreModel: types.optional(RequirementTemplateStoreModel, {}),
```

Comment:
why are these two pascal case? They are instances so should be camel case

Reply:
fixed